python/ray/actor.py

```diff
@@ -458,21 +494,30 @@ def _remote(self,
             actor_method_cpu = ray_constants.DEFAULT_ACTOR_METHOD_CPU_SPECIFIED
 
         function_name = "__init__"
-        function_descriptor = FunctionDescriptor(
-            meta.modified_class.__module__, function_name,
-            meta.modified_class.__name__)
+        if meta.is_cross_language:
+            function_descriptor = meta.function_descriptor
+            module_name = ""
+        else:
+            function_descriptor = PythonFunctionDescriptor(
+                meta.modified_class.__module__, function_name,
+                meta.modified_class.__name__)
+            module_name = meta.modified_class.__module__
 
         # Do not export the actor class or the actor if run in LOCAL_MODE
         # Instead, instantiate the actor locally and add it to the worker's
         # dictionary
         if worker.mode == ray.LOCAL_MODE:
+            if meta.is_cross_language:
+                raise Exception(
```

Comment:
Exception -> NotImplementedError ?

Reply:
Cross language feature in local mode won't be implemented. Maybe Exception is better than NotImplementedError?